backend/lm_backend/api/models/feature.py

```diff
@@ -39,13 +39,12 @@ def booked_total(self) -> int:
         """
         return sum(b.quantity for b in self.bookings)
 
-    @booked_total.inplace.expression
-    @classmethod
-    def _booked_total_expression(cls) -> SQLColumnExpression[int]:
+    @hybrid_property
+    def product_name(self) -> int:
```

Comment:
Why is this one needed when we have the product_name returned by the query?

Reply:
The idea was to make it possible to use the `FlatFeatureSchema` with a single model instance as well as the data brought back by the `read_all()` method. I guess we aren't using it at the moment.

Reply:
Removed in commit 8eb6c76